samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala

```diff
@@ -21,23 +21,20 @@ package org.apache.samza.checkpoint.kafka
 
 import java.nio.ByteBuffer
 import java.util
-import java.util.Properties
-
-import kafka.api._
-import kafka.common.{ErrorMapping, InvalidMessageSizeException, TopicAndPartition, UnknownTopicOrPartitionException}
-import kafka.consumer.SimpleConsumer
-import kafka.message.InvalidMessageException
-import kafka.utils.ZkUtils
+import java.util.{Collections, Properties}
 
+import _root_.kafka.consumer.SimpleConsumer
```

Comment:
Why do we still need the `SimpleConsumer` import here?

Reply:
cleaned up.